### PR TITLE
Ryuk story and detections

### PR DESCRIPTION
**Pull request by josehelps**
adds a new story:
- [x] Ryuk Ransomware

adds new detections:
- [x] Ryuk Test Files Detected
- [x] Windows Security Account Manager Stopped
- [x] Windows DisableAntiSpyware Registry
- [x] Windows connhost exe started forcefully

improved `validate.py` by adding additional logging

### CODE REVIEW

**Review by rosplk — CHANGES_REQUESTED** [review]
Looks good to The search looks for files that contain the key word *Ryuk* under any folder in the C drive, which is consistent with Samsam  ##changeto'ryuk'## propagation. how_to_implement: You must be ingesting data that records the file-system activity' this needs to be fixed this is Ryuk Test Files Detected

File: bin/generate.py [hunk 1]
```diff
@@ -279,9 +279,9 @@ def generate_workbench_panels(response_tasks, stories, TEMPLATE_PATH, OUTPUT_PAT
                 template = j2_env.get_template('panel.j2')
                 file_path = "default/data/ui/panels/workbench_panel_" + response_file_name + ".xml"
                 output_path = path.join(OUTPUT_PATH, file_path)
-                if response_task['search'].find(">") is not -1:
+                if response_task['search'].find(">") != -1:
                     response_task['search']= response_task['search'].replace(">","&gt;")
```
Comment (ghoto) [inline bin/generate.py]: I think the "if"'s here are redundant. 
```
response_task['search']= response_task['search'].replace(">","&gt;")
```
is going to replace ">" if it exists and leave it untouched if it doesn't.

**Review by ghoto — COMMENTED** [review]
LGTM, small comment.

File: bin/generate.py (hunk 1 above)
Comment (josehelps) [reply bin/generate.py]: oh good point 😄 fixing.

**Review by rosplk — DISMISSED** [review]

**Pull request merged**